e2e/pom.xml

```diff
@@ -14,12 +14,12 @@
         <dependency>
             <groupId>org.dhatim</groupId>
             <artifactId>fastexcel</artifactId>
-            <version>0-SNAPSHOT</version>
+            <version>0.12.12</version>
```

Comment:
We should keep `0-SNAPSHOT` to use the local artifact.

Reply:
versions reverted